velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -409,14 +415,15 @@ class ExpressionFuzzer {
   }
 
   void appendConjunctSignatures() {
```

Comment:
As discussed offline, we would like to control the probability of generating conjunct in expressions. Appending conjunct signatures to `signatures_` wouldn't allow this control. Do you plan to make the change in a separate PR?

Reply:
Thanks @kagamiori  for helping with reviweing. Yeah that should be belong to another PR.